api/handlers.go

```diff
@@ -52,9 +52,14 @@ func (h *handler) getAllUnitsHandler(w http.ResponseWriter, _ *http.Request) {
 // /api/v1/system/health/units/:unit_id:
 func (h *handler) getUnitByIDHandler(w http.ResponseWriter, r *http.Request) {
 	vars := mux.Vars(r)
-	unitResponse, err := h.monitoringResponse.GetUnit(vars["unitid"])
+	unitId := vars["unitid"]
```

Comment:
just wondering why this was pulled out into a var (doesn't look to be used elsewhere)

Reply:
Accidentally committed my WIP version. Thanks for pointing out.